README.md

```diff
@@ -39,22 +34,22 @@ This release is supported by the next SDK versions
 
 - azure-iot-sdk-c [public-preview](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp) branch
 - NPM package for node [azure-iot-digitaltwins-device@1.0.0-pnp-refresh.0](https://www.npmjs.com/package/azure-iot-digitaltwins-device/v/1.0.0-pnp-refresh.0)
-- Python [Python DEvice SDK for the BugBash](https://aka.ms/PythonDevicePnP0508) 
-- azure-iot-device [TBD](https://pipy.com/) pip package
+- Python [Python Device SDK for the BugBash](https://aka.ms/PythonDevicePnP0508) 
 
 ##### Service SDK
 
 - NPM package for node [azure-iot-digitaltwins-service@1.0.0-pnp-refresh.0](https://www.npmjs.com/package/azure-iot-digitaltwins-service/v/1.0.0-pnp-refresh.0)
-- Python [Python Service SDK for the BugBash](https://aka.ms/PythonServicePnP0508) 
-- PIP package for python TBD for PPR
+- Python [Python Service SDK for the BugBash](https://aka.ms/PythonServicePnP0508)
 
 #### PnP related tools
 
 The tools are available as internal previews:
 
+- [IoT Model Repository](https://test.iotmodels.trafficmanager.net/)
 - Azure IoT Explorer (available as private release in [YingXue/azure-iot-explorer/releases](https://github.com/YingXue/azure-iot-explorer/releases) )
 - VSCode extension for DTDL (availabe as a private release in [microsoft/vscode-azure-digital-twins/releases](https://github.com/microsoft/vscode-azure-digital-twins/releases) ). To install, download the `.vsix` file and install in VSCode as decribed [here](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix).
-- [IoT Model Repository](https://test.iotmodels.trafficmanager.net/)
+
+>Note: The VSCode extension does not support semantic types in this release. Any semantic type will be ignored (not validated) and there is no intellisense support.
 
 ## Sample Solution  Features
```

Comment:
When referencing common interfaces, please mention that these should be implemented already by the device SDKs as sample code (it is in C at least and Node IIRC) so this would save bug-basher time from needing to re-implement.  They just need to compose.

Reply:
Ahh - I see you have a `Note: You can reuse the implementation of DeviceInformation and SDKInformation from the existing samples` way below.  You can ignore my comment here, though I'd still recommend as only a nit calling out sooner that there are samples that exist already so basher doesn't think they have to do all this.

Reply:
reworded